packages/x-tree-view/src/internals/plugins/useTreeViewFocus/useTreeViewFocus.ts

```diff
@@ -96,7 +96,11 @@ export const useTreeViewFocus: TreeViewPlugin<UseTreeViewFocusSignature> = ({
       const itemElement = document.getElementById(
         instance.getTreeItemIdAttribute(state.focusedItemId, itemMeta.idAttribute),
       );
+
       if (itemElement) {
+        if (instance.isItemBeingEdited(state.focusedItemId)) {
```

Comment:
:grimacing: Not sure how we can support `SimpleTreeView` that do not have this plugin.

I think I'll try again to add a notion of `optionalDependencies` where all the methods are nullable.
That way the plugins used in `SimpleTreeView` can have the label editing plugin as optional and test for the existence of the methods.

Reply:
Exploration in #13523